web/src/containers/activity/CostAllocateFFP.js

```diff
@@ -67,14 +65,18 @@ class CostAllocateFFP extends Component {
                     onChange={this.handleFFP(year)}
                   />
                   <div className="lg-flex">
-                    {allocations.map(({ id, amount }) => (
```

Comment:
Made the list of allocations explicit instead of being generic.  It's not like we'll be splitting costs between more than state and federal any time soon.  Who'd even be the other partner?

Reply:
Oh, this is excellent. Yay for explicit variable names!